app/helpers/foreman_openscap_helper.rb

```diff
@@ -15,10 +15,15 @@ def scap_doc_url(section = '')
     documentation_url(section, root_url: scap_root_url)
   end
 
+  def doc_flavor
+    ::Foreman::Plugin.installed?('katello') ? 'katello' : 'foreman'
```

Comment:
What if we add check-for-katello-plugin helper? Something like 
https://github.com/theforeman/foreman_openscap/blob/master/lib/foreman_openscap/engine.rb#L303-L305

Reply:
Huh, we already have that https://github.com/theforeman/foreman_openscap/blob/master/lib/foreman_openscap/engine.rb#L2 ?